ansible/ansible.cfg

```diff
@@ -0,0 +1,6 @@
+[defaults]
+inventory = hosts
+private_key_file = ~/.ssh/nhs-england-data.pem
```

Comment:
Are we sure we want this committed?
I have a different `./ansible/ansible.cfg` (hence gitignored...)

And the name of the .pem file used for say production deploys is different ...

Reply:
Having a project level one is required for the `pipelining` section.

Having `private_key_file` here removes some user configuration (I had to add a section in the hosts file for each host to point to the pem file) and `inventory` makes the command to update a little shorter (not a huge win, but less to think about I found).

What do you have in your local copy?

Reply:
```
[ssh_connection]
ssh_args = 
```
but pointing to a different Pem file (e.g. the one we use for production deploy)